onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -2063,6 +2063,21 @@ def aten_copy(
     return self
 
 
+@torch_op("aten::_to_copy", trace_only=True)
+def aten__to_copy(
+    self: TTensor,
+    dtype: int = -1,
+    non_blocking: bool = False,  # pylint: disable=unused-argument
+) -> TTensor:
+    """_to_copy(Tensor self, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, bool non_blocking=False, MemoryFormat? memory_format=None) -> Tensor"""
+
+    if dtype == -1:
+        self = op.Identity(self)
+    else:
+        self = op.Cast(self, to=dtype)
+    return self
```

Comment:
```suggestion
    if dtype != -1:
        return common_ops.cast_to(a, dtype)
    return self
```

Based on https://github.com/microsoft/onnxscript/pull/1147